package.json

```diff
@@ -18,7 +18,9 @@
   "bugs": {
     "url": "https://github.com/primer/view_components/issues"
   },
-  "files": ["app/components/primer/*.js"],
+  "files": [
+    "app/components/primer/*.js"
+  ],
```

Comment:
Do we know why this format keeps moving around?

Reply:
This is `npm version --no-git-tag-version ...` fault, so I think it is better to leave it this way 😅